src/MCT.js

```diff
@@ -210,6 +210,31 @@ define([
     };
 
     /**
+     * Inject build information for the About dialog et al.
+     * @private
+     */
+    MCT.prototype.specifyBuildInfo = function (buildInfo) {
```

Comment:
Don't like this being on the prototype.  Can we just throw this in the openmct.js file, i.e `if (BUILD_INFO) { openmct.legacyExtension(...) }`?  

Or if you really want to test it, hide it in the MCT constructor?

Reply:
How about hiding it in a build info plugin? Agree that `MCT.prototype` is not the best place for this, but also don't like a lot of low-level in top-level `openmct.js`. Can relocate those tests easily

Reply:
@VWoeltjen that works.